templates/themes/maison/full/store.html.twig

```diff
@@ -0,0 +1,146 @@
+        <section class="section section--standard mt-10">
+            <div class="container">
+                <div class="row">
+                    <div class="col-lg-6 text-center pr-lg-0">
+                        <div class="px-lg-5 px-md-4 py-3 bg-grey-very-light rounded-left-2 h-100">
+                            <p class="font-weight-bold">Store Facts</p>
+                            <div class="d-flex justify-content-between">
+                                <div class="d-flex d-inline-flex align-items-center">
+                                    <p class="heading-sm mr-1 mb-0">375</p>
+                                    <p class="mb-0">Professionals</p>
+                                </div>
+                                <div class="d-flex d-inline-flex align-items-center">
+                                    <p class="heading-sm mr-1 mb-0">375</p>
+                                    <p class="mb-0">Products</p>
+                                </div>
+                                <div class="d-flex d-inline-flex align-items-center">
+                                    <p class="heading-sm mr-1 mb-0">{{ ez_render_field(content, 'floors') }}</p>
+                                    <p class="mb-0">Floors</p>
+                                </div>
+                                <div class="d-flex d-inline-flex align-items-center">
+                                    <p class="heading-sm mr-1 mb-0">{{ ez_render_field(content, 'storeArea') }}</p>
+                                    <p class="mb-0">SF</p>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="col-lg-6 text-center pl-lg-1 mt-4 mt-lg-0">
+                        <div class="px-4 py-3 bg-grey-very-light rounded-right-2 h-100">
+                            <p class="font-weight-bold">Store Categories</p>
+                            <div class="d-flex flex-wrap justify-content-between ">
+                                {% for i in 1..4 %}
+                                    <a href="" class="btn btn--sm btn--white mb-1 {{ not loop.last ?  'mr-1' : '' }}">
+                                        Living Room
+                                    </a>
+                                {% endfor %}
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </section>
+        <section class="section section--standard my-10">
+            <div class="container">
+                <div class="row mt-">
+                    <div class="col-lg-4 col-md-12">
+                        <h4 class="heading-lg">Find a inspirations and ideas to help you with</h4>
+                        <h5 class="heading-sm mt-4">delivery, assembling, painting, renovation, etc.</h5>
+                        <a href="/inspirations" class="btn btn--lg btn--blue mt-5">Find Inspirations</a>
```

Comment:
Not a big fan of having this hardcoded link. I understand that it's a demo but we should also show best practices.
```suggestion
                        <a href="/inspirations" class="btn btn--lg btn--blue mt-5">{{ 'Find Inspirations' | trans }}</a>
```